homeassistant/components/environment_canada/sensor.py

```diff
@@ -79,6 +71,8 @@ def validate_station(station):
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)):
```

Comment:
Remove monitor conditions in favor of ADR0004

Reply:
Thanks, I’ve looked at ADR0004 but I’m not sure how it applies to the section you’ve pointed to. This list allows the user to specify what sensors they would like to create or not.

Reply:
Sorry, my wrong -> https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md 0003 :D

Reply:
Got it, makes sense. I’ll make the switch.